dnnv/nn/operations/tensor.py

```diff
@@ -256,6 +273,55 @@ def from_onnx(cls, onnx_node, *inputs):
         return cls(*inputs, axes=axes, name=onnx_node.name)
 
 
+class ReduceL2(Operation):
+    def __init__(self, x, axes, keepdims, *, name: Optional[str] = None):
+        super().__init__(name=name)
+        self.x = x
+        if axes is None:
+            self.axes = None
+        elif isinstance(axes, int):
+            self.axes = axes
+        elif len(axes) == 1:
+            self.axes = int(axes[0])
+        elif len(axes) > 1:
+            self.axes = tuple(axes)
+        self.keepdims = keepdims
+
+    @classmethod
+    def from_onnx(cls, onnx_node, *inputs):
+        attributes = {a.name: as_numpy(a) for a in onnx_node.attribute}
+        axes = attributes.get("axes")
+        keepdims = attributes.get("keepdims")
+        return cls(*inputs, axes=axes, keepdims=keepdims, name=onnx_node.name)
+
+
+class Squeeze(Operation):
+    def __init__(self, x, axes, *, name: Optional[str] = None):
+        super().__init__(name=name)
+        self.x = x
+        self.axes = axes
+
+    @classmethod
+    def from_onnx(cls, onnx_node, *inputs):
+        attributes = {a.name: as_numpy(a) for a in onnx_node.attribute}
+        axes = attributes.get("axes")
+        return cls(*inputs, axes=axes, name=onnx_node.name)
+
+
+class Upsample(Operation):
```

Comment:
Could you add a comment stating that this operation is deprecated. Maybe it should also give a warning, but I think a comment is sufficient for now.

```suggestion
class Upsample(Operation):
    ```DEPRECATED```
```